### ISSUE TITLE
Some Private Keys not detected

### ISSUE THREAD

**Issue by dxa4481** [trufflesecurity/trufflehog#308]
**Describe the bug**
Our test key repo has a key that's not being found https://github.com/trufflesecurity/test_keys

**To Reproduce**
Run the scanner against the test key repo, no private key shows up:
```
docker run -it -v "$PWD:/pwd" trufflesecurity/trufflehog:latest github --repo https://github.com/trufflesecurity/test_keys
```

**Expected behavior**
Private key should be found

**Comment by bolshoytoster** [comment]
@dxa4481 weidly enough, it detects the openssh key if I clone the repo and run `trufflehog filesystem --directory=test_keys`.

`trufflehog git file://test_keys` gives the same output as with `github`, so the issue is likely with [Git.ScanRepo](https://github.com/trufflesecurity/trufflehog/blob/b85467e556f835f15a4755a2c984e30d13771d56/pkg/sources/git/git.go#L380-L397).

The `Basic auth` and `AWS base64 encoded` aren't detected either way though, so that's also an issue.

**Comment by dustin-decker** [comment]
This should be fixed in v3.2.3. Thanks @bill-rich !
